StackOverflow/Controller/QuestionsCell.m

```diff
@@ -0,0 +1,24 @@
+//
+//  QuestionsCell.m
+//  StackOverflow
+//
+//  Created by Mali on 2017/10/04.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+#import "QuestionsCell.h"
+
+@implementation QuestionsCell
```

Comment:
The logic for setting the contents of a cell should be contained in the TableViewCell subclass. If you need me to explain what I mean here please let me know. This is a common error that many developers make. The code for configuring the content of TableViewCells is contained in the UITableViewDataSource which is incorrect for a number of reasons. Ideally you should define a single setter on the TableViewCell subclasses that takes the model that the TableViewCell subclasses represents and in that setter all the config of the TableViewCell should be done.

Reply:
Logic added in the TableViewCell